src/styles/main.scss

```diff
@@ -1 +1,113 @@
-// put code here
+
+html {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  height: 100%;
+}
+
+.clock {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+
+  position: relative;
+  width: 500px;
+  height: 500px;
```

Comment:
Need to fix
<img width="282" alt="image" src="https://github.com/mate-academy/layout_stop-watch/assets/64750958/baaa5e16-0a09-461a-8352-5df24213ae9c">